### ISSUE TITLE
More streamlined database initialization

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster-backend#566]
It should be possible to initialize the database without running a finicky Perl one-liner to figure out where the SQL file was installed. This is currently a rough edge with regard to administration and installation testing.

A conceivable solution to this is to put a new executable in the path that locates the SQL file and pumps it to the appropriate SQL client. A command line parameter could be used for selecting DB engine.

**Comment by pnax** [comment]
> It should be possible to initialize the database without running a finicky Perl one-liner to figure out where the SQL file was installed. 

I think that this part of this issue has been addressed with #834.

> A conceivable solution to this is to put a new executable in the path that locates the SQL file and pumps it to the appropriate SQL client. A command line parameter could be used for selecting DB engine. Such a tool could also be hooked up to the clean-up SQL files.

This second part is asking for an administrative tool that can be used. This can be a good idea. However I think there are already a lot of things on the roadmap. Could we postpone this issue to a later release (or delay it until further notice) ?

**Comment by matsduf** [comment]
I do not think that running the script that creates the database is much of a problem. It is easy to create the instruction and it easy to run. I think we should put our efforts on something else.
